src/client.js

```diff
@@ -144,4 +152,6 @@ function init(options = {}) {
   return new MondayClientSdk(options);
 }
 
+initBackgroundTracking(init());
```

Comment:
I need be inside the constructor.
Now it will ignore options such as "withoutScrollHelper"